packages/volto/src/components/manage/Controlpanels/Addons/AddonsControlpanel.tsx

```diff
@@ -0,0 +1,356 @@
+              content={intl.formatMessage(messages.addonNotInstalled)}
+            />,
+          );
+        })
+        .finally(() => dispatch(listAddons() as AnyAction));
+    },
+    [dispatch, title, intl],
+  );
+
+  const onUninstall = useCallback(
+    (event: PressEvent) => {
+      const value = event.target.id.replace('installed-', '');
+      dispatch(uninstallAddon(value) as AnyAction)
+        .then(() => {
+          toast.success(
+            <Toast
+              success
+              title={intl.formatMessage(messages.success)}
+              content={intl.formatMessage(messages.addonUninstalled)}
+            />,
+          );
+        })
+        .catch(() => {
+          toast.error(
+            <Toast
+              error
+              title={intl.formatMessage(messages.error)}
+              content={intl.formatMessage(messages.addonNotUninstalled, {
+                title: title,
+              })}
+            />,
+          );
+        })
+        .finally(() => dispatch(listAddons() as AnyAction));
+    },
+    [dispatch, intl, title],
+  );
+
+  const onUpgrade = useCallback(
+    (event: PressEvent) => {
+      const value = event.target.id.replace('upgradable-', '');
+      dispatch(upgradeAddon(value) as AnyAction)
+        .then(() => {
+          toast.success(
+            <Toast
+              success
+              title={intl.formatMessage(messages.success)}
+              content={intl.formatMessage(messages.addonUpgraded)}
+            />,
+          );
+        })
+        .catch(() => {
+          toast.error(
+            <Toast
+              error
+              title={intl.formatMessage(messages.error)}
+              content={intl.formatMessage(messages.addonNotUpgraded)}
+            />,
+          );
+        })
+        .finally(() => dispatch(listAddons() as AnyAction));
+    },
+    [dispatch, intl],
+  );
+
+  return (
+    <div id="page-addons" className="ui container controlpanel-addons">
+      <Helmet title={intl.formatMessage(messages.addOns)} />
+      <div className="main-section">
+        <h1>
+          <FormattedMessage
+            id="Add-ons Settings"
+            defaultMessage="Add-ons Settings"
+          />
+        </h1>
+
+        {loadingAddons ? (
+          <Dimmer active>
+            <Loader />
+          </Dimmer>
+        ) : (
+          <>
+            <div>
+              <h2>
+                <FormattedMessage
+                  id="Activate and deactivate"
+                  defaultMessage="Activate and deactivate add-ons in the lists below."
+                />
+              </h2>
+              <FormattedMessage
+                id="Add Addons"
+                defaultMessage="To make new add-ons show up here, add them to your configuration, build, and restart the server process. For detailed instructions see"
+              />
+              &nbsp;
+              <a
+                href="https://6.docs.plone.org/install/"
```

Comment:
Hold on this. We are in the process of updating the documentation, and the new page is not yet released. See https://github.com/plone/volto/pull/6397/files#diff-5ac1c30eabae6878f93007dc64b3129cc05eaa5f878cd274fc27779edd410276

```suggestion
                href="https://6.docs.plone.org/volto/development/add-ons/install-an-add-on.html"
```